scania-digital-design-system/tegel
# feat(icons): icons prefix change

**Describe pull-request**  
Changing prefix in icons package/folder and in icon component

**Solving issue**  
Fixes: DTS-1694

**How to test**  
1. Check if the Gulp file in the icon package/folder has tds-icon as a prefix
2. Check if native and web component icon stories have tds as prefix 
3. Check if other components have native or web component with new prefix in place

**Suggested test steps**


**Additional context**  
Web font story will appear broken as new font files need upload to new CDN solution for Tegel icons.
